src/JustSaying/AwsTools/MessageHandling/SqsMessagePublisher.cs

```diff
@@ -104,3 +133,11 @@ public InterrogationResult Interrogate()
         });
     }
 }
+
+public sealed class PublishCompressionOptions
+{
+    public static PublishCompressionOptions Default => new();
```

Comment:
Did you mean this to be `=>` and a new one every time, or did you mean `{ get; } = new()`? For the former the name seems a bit misleading to me, as it suggests a shared static object for re-use like `JsonSerializerOptions.Default`. If the latter, then is it a problem that people can mutate the values?